.github/workflows/alert_unsigned_commits.yml

```diff
@@ -0,0 +1,44 @@
+name: Alert Unsigned Commits
+on:
+  pull_request_review:
+    types: [submitted]
+  pull_request:
+    types: [reopened, synchronize]
+
+permissions:
+  pull-requests: write
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  alert-unsigned:
+    if: github.event.review.state == 'approved'
+    name: Alert Unsigned Commits
+    runs-on: ubuntu-latest
+    steps:
+      - name: Alert Unsigned Commits
+        run: |
+          env | grep -i GIT | sort  # TODO remove before merge
+          set -x
+          HEAD_SHA=${{ github.event.pull_request.head.sha }}
+
+          BRAVE_CORE_PATH=brave/brave-core
+          post_once() {
+              comments=$(gh -R "$BRAVE_CORE_PATH" pr view "$GITHUB_HEAD_REF" --json comments --jq ".comments[].body")
+              grep -q "$(head -1 <<<"$1")" <<<"$comments" || gh -R "$BRAVE_CORE_PATH" pr comment "$GITHUB_HEAD_REF" -b "$1"
```

Comment:
When grepping for plain strings, rather than regular expressions, `-F` should be passed

Reply:
Done